gravitee-rest-api-management/gravitee-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java

```diff
@@ -177,20 +179,17 @@ private Response getImageResponse(final Request request, final String api, Inlin
     }
 
     @POST
-    @ApiOperation(
-            value = "Manage the API's lifecycle",
-            notes = "User must have the MANAGE_LIFECYCLE permission to use this service")
-    @ApiResponses({
-            @ApiResponse(code = 204, message = "API's picture"),
-            @ApiResponse(code = 500, message = "Internal server error")})
+    @Operation(summary = "Manage the API's lifecycle", description = "User must have the MANAGE_LIFECYCLE permission to use this service")
+    @ApiResponse(responseCode = "204", description = "API's picture")
+    @ApiResponse(responseCode = "500", description = "Internal server error")
     @Permissions({
             @Permission(value = RolePermission.API_DEFINITION, acls = RolePermissionAction.UPDATE)
     })
     public Response doApiLifecycleAction(
             @Context HttpHeaders headers,
-            @ApiParam(required = true, allowableValues = "START, STOP")
+            @Parameter(required = true)
```

Comment:
using:
```java
    @Parameter(required = true, schema = @Schema(allowableValues = "START" ))
    @QueryParam("action") LifecycleAction action
```

will result in:

```yaml
      - name: action
        in: query
        required: true
        schema:
          type: string
          enum:
          - START
          - STOP
          - START
```

So we cant use it.
Since `START` and `STOP` are the only values for `LifecycleAction` we don't need to restrict the values.

Is there any reason why `LifecycleActionParam` is used instead of `LifecycleAction` directly?
I'm going to replace the `*Params` classes for Enums by the respective Enums.

Reply:
https://github.com/swagger-api/swagger-core/issues/2949